dashboard/test/models/pd/workshop_test.rb

```diff
@@ -1,3 +1,4 @@
+require_relative '../../../../shared/test/spy_newrelic_agent'
```

Comment:
I couldn't find an example of a shorter way to write this, though it might be possible with an absolute path?

Reply:
Huh... I'm not seeing an example of an absolute-path import from our shared directory.